views/confirm.hbs

```diff
@@ -28,20 +27,20 @@
 </head>
 
 <body>
-    <!-- <div class='wrapper'> -->
-    <div class='confirm'>
-        <div class="thankyou">
-            <h1>Thank you for signing up!</h1>
+    <div class='wrapper'>
+
+        <div class='confirm'>
+            <div class="thankyou">
+                <h1>Thank you for signing up!</h1>
+            </div>
+            <p class='text'>Please check your inbox (and spam) and click on the link we sent to verify your account.</p>
+            <p class='retry-text'>Think it's taking too long? .edu emails often have some delay to them, but feel free to have us try again.</p>
+            <button>RESEND EMAIL</button>
```

Comment:
So I don't know how long it's been like this, but turns out this button currently doesn't do anything (and didn't last year either). If you check out the JS that handles it, the button needs the `.resend-verification` class for the event handler to register properly. Be sure to test it one more time once you fix this

Reply:
oh no, that's fun. Well the button works now!